accelerator-index-retrieve-resumes-sql/chat-with-documents-sql/workflow.json

```diff
@@ -0,0 +1,104 @@
+            "type": "ServiceProvider",
+            "description": "Use this query to retrieve vector embeddings from your SQL vector table. Update the query based on your SQL table and schema",
+            "inputs": {
+                "parameters": {
+                    "query": "SELECT TOP(2) filename, chunkid, chunk,\n           1-vector_distance('cosine', CAST(@embedding AS VECTOR(1536)), embedding) AS similarity_score,\n           vector_distance('cosine', CAST(@embedding AS VECTOR(1536)), embedding) AS distance_score\n    FROM dbo.resumedocs\n    ORDER BY distance_score ",
+                    "queryParameters": {
+                        "embedding ": "@{body('Azure_OpenAI_-_Get_an_embedding')['embedding']}"
+                    }
+                },
+                "serviceProviderConfiguration": {
+                    "connectionName": "azuresql_#workflowname#",
+                    "operationId": "executeQuery",
+                    "serviceProviderId": "/serviceProviders/sql"
+                }
+            },
+            "runAfter": {
+                "Azure_OpenAI_-_Get_an_embedding": [
+                    "SUCCEEDED"
+                ]
+            }
+        },
+        "Azure_OpenAI_-_Get_chat_completions": {
+            "type": "ServiceProvider",
+            "description": "Update the system message for the LLM based on your requirements",
+            "inputs": {
+                "parameters": {
+                    "deploymentId": "@parameters('OpenAIChatModel_#workflowname#')",
+                    "messages": [
+                        {
+                            "role": "System",
+                            "content": "You are an intelligent & funny assistant who will exclusively answer based on the data provided in the search_results:\n- Use the information from search_results to generate your top 3 responses. If the data is not a perfect match for the user's query, use your best judgment to provide helpful suggestions and include the following format:\n  File: {filename}\n  Chunk ID: {chunkid}\n  Similarity Score: {similarity_score}\n  Add a small snippet from the Relevant Text: {chunktext}\n  Do not use the entire chunk\n- Avoid any other external data sources.\n- Add a summary about why the candidate maybe a goodfit even if exact skills and the role being hired for are not matching , at the end of the recommendations. Ensure you call out which skills match the description and which ones are missing. If the candidate doesnt have prior experience for the hiring role which we may need to pay extra attention to during the interview process.\n- Add a Microsoft related interesting fact about the technology that was searched \n\nsearch_results = @{body('SQL_Server_-_Vector_Search')}"
+                        },
+                        {
+                            "role": "User",
+                            "content": "@triggerBody()?['question']",
+                            "user": "Divya"
```

Comment:
Is this intentional? If possible, we may want to consider removing personal information

Reply:
@divyaswarnkar: No personal info should appear. You can use the CELA fictitious name generator:

https://microsoft.sharepoint.com/sites/celaweb-tools/sitepages/fictitiousnamefinder.aspx